### DISCUSSION TITLE
交了两年的钱，没有收到邮件，也无法授权其他设备

### DISCUSSION THREAD

**Discussion by icingjingjing** [getlantern/lantern-forum-cn#743] [Q&A]
交了两年的钱，且使用了朋友给的邀请码，但是存在如下三个问题：
1、邮箱没有收到任何邮件，使用的@qq.com的邮箱。
2、没有找到“给其他设备授权”的菜单。
3、使用了朋友给的邀请码，但是并没有多赠送1个月。
![lantern付款](https://github.com/user-attachments/assets/83659cef-18c4-4a24-a780-64d14e9c84dd)
![3](https://github.com/user-attachments/assets/d748429e-42e3-4758-96a5-0bcd5890cb31)

**Comment by MaxMKMax** [comment]
@icingjingjing 您好. 好的, 我会帮您提交问题上去. 谢谢您的反馈.

**Comment by MaxMKMax** [comment]
2. 给其他设备授权在右手边, 点进去设定就可以了. 
https://lantern.io/zh-Hans/faq

**Reply by icingjingjing** [reply]
授权其他电脑这个，明天我再试试。

**Comment by MaxMKMax** [comment]
@icingjingjing 1. 请问您指的没有收到邮件是指没有收到激活码对吗? 我们一般不发邮件的, 除了激活码. 您可以登入吗?

**Reply by icingjingjing** [reply]
我只有使用旧版本6.10.7的登录上了。其他的都不行。
另外，授权其他设备，如我问题中的截图，没有看到对应的菜单项

**Reply by icingjingjing** [reply]
而且，一晚上只有刚才一两分钟能上google，感觉不像是成功升级到了专业版本

**Comment by MaxMKMax** [comment]
@icingjingjing 请您提供我账号邮箱. 这样才帮您查看.

**Reply by icingjingjing** [reply]
158163215@qq.com
另外是从付款后，没收到任何邮件，是正常的吗

**Reply by icingjingjing** [reply]
现在主要是1、这个专业版确认生效了不。

**Reply by icingjingjing** [reply]
2、少了一个月的赠送。

**Comment by MaxMKMax** [comment]
@icingjingjing 您好. 您的这个版本太旧了. 肯定用不了. 您重新下载最新版本吧.

少送一个月我会帮您申请.

您需要用最新版本, 我这里也帮您问看您是否需要激活码

**Reply by icingjingjing** [reply]
1、8.0.0英文版，用邮箱登录时会报错：LateInitializationError: Field 'methodChannel' has not been initialized
2、8.0.0中文版，输入邮箱后，点按钮没任何反应。
3、用7.8.6版本，程序启动后几秒就自己退出了。
哪个能用啊
